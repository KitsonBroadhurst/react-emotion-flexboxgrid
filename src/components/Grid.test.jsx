/* eslint-env jest */
/* eslint-disable no-unused-expressions */

import React from 'react';
import { shallow } from 'enzyme';

import Grid from './Grid';

describe('Grid', () => {
  it('should work', () => {
    expect(<Grid />).toBeTruthy();
    expect(shallow(<Grid />)).toMatchSnapshot();
  });
});
