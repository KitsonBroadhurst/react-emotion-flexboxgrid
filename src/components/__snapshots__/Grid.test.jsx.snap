// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid should work 1`] = `
.emotion-0 {
  margin-right: auto;
  margin-left: auto;
}

@media only screen and (min-width:48em) {
  .emotion-0 {
    width: 46rem;
  }
}

@media only screen and (min-width:64em) {
  .emotion-0 {
    width: 61rem;
  }
}

@media only screen and (min-width:75em) {
  .emotion-0 {
    width: 76rem;
  }
}

<div
  className="emotion-0 emotion-1"
/>
`;
